packages/discovery-react-components/src/components/SearchResults/components/SpellingSuggestion/SpellingSuggestion.tsx

```diff
@@ -10,9 +10,16 @@ interface SpellingSuggestionProps {
    * Message prefix used when displaying spelling suggestion
    */
   spellingSuggestionPrefix?: string;
+  /**
+   * Used for tracking analytics
```

Comment:
i would update this to match the javadoc you have in the other ones

Reply:
Must have missed that one. Updated in latest commit.